department-of-veterans-affairs/vets-website
# Update service number default label text

## Summary
Update `serviceNumberUI()` web component pattern default label text to match VA design system form label standard. 
Please see guidance under Service History on the Form Labels page:
[https://design.va.gov/content-style-guide/form-labels](https://design.va.gov/content-style-guide/form-labels)

Forms using service number web component pattern:
- 21P-527EZ
- 21-0972
- 21-0845

## Related issue
[Pension Benefits - Service Period](https://app.zenhub.com/workspaces/benefits-pension-64e775aaa6b1ca1ed49b2ede/issues/gh/department-of-veterans-affairs/va.gov-team/68564)

## How to verify
- On pension benefits application Step 2 Service History, verify that the label is 'Military Service number'
- On Sign VA claim forms as an alternate signer application, verify that the label is overridden to be 'Veteran's service number'

## Screenshots

| Default Label Text | Overridden Label Text |
| ------- | ------- |
|![localhost_3001_pension_application_527EZ_](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/95036f00-516c-473c-9be5-75d34b9af1fe)|![localhost_3001_supporting-forms-for-claims_alternate-signer-form-21-0972_](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/2d2049e9-10ab-41fc-8680-5571e13e0a85)


## What areas of the site does it impact?
- 21P-527EZ
- 21-0972
- 21-0845
And future use cases of service number web component pattern

### Quality Assurance & Testing


### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
